app/components/post_draft/draft_input/draft_input.tsx

```diff
@@ -260,6 +260,7 @@ function DraftInput({
                             disabled={sendActionDisabled}
                             sendMessage={handleSendMessage}
                             showScheduledPostOptions={handleShowScheduledPostOptions}
+                            scheduledPostEnabled={scheduledPostsEnabled}
```

Comment:
Looking at the code, it seems that we allow long press in the send button when we are not licensed. That is not correct, right?

You should create a `observeScheduledPostEnabled` function that checks both the config and the license, and use it in this file index, so it propagates properly to all the relevant places.

While at it, you should consider do something similar on `app/actions/remote/scheduled_post.ts` on `fetchScheduledPosts`, `app/screens/global_drafts/index.ts` and `app/screens/home/channel_list/categories_list/index.ts`,  since you also do a check only on the config and not the license there.

Reply:
Thanks @larkox for pointing this out. This is a nice catch. Updated the code.